.travis.yml

```diff
@@ -7,14 +9,17 @@ php:
   - 7.0
   - 7.1
   - nightly
-  - hhvm
+  - hhvm-3.6
 
 matrix:
   allow_failures:
     - php: nightly
 
 env:
-  - CUBRID_VERSION=9.3.0/CUBRID-9.3.0.0206 CUBRID_PDO_VERSION=9.3.0.0001
+  global:
+    - CUBRID_VERSION=9.3.0/CUBRID-9.3.0.0206
+    - CUBRID_PDO_VERSION=9.3.0.0001
+    - secure: Q3dD9eL9yGDbMbdnNeLPSruRmZH515tLOTBam5rnsqX8spE+/HLqskLwUKIX78ansVdhu/OLGO72rBHvUotrQPLjqp6GnOSXw15gaz1LcxzfqbiZSbNvYw+7nl0hUjysLEMfZFEGWqrrk8BMuRAWyxABCrdR/91CyTFJhhcTnpg=
```

Comment:
This is an encrypted token from my account. I guess it's better to replace it with token of someone from core developers.